reana_server/utils.py

```diff
@@ -160,16 +161,6 @@ def mv_workflow_files(source: str, target: str) -> None:
         shutil.move(os.path.join(source, entry), target)
 
 
-# FIXME: use `is_relative_to` from the standard library when moving to Python 3.9
-def is_relative_to(path: pathlib.Path, base: pathlib.Path) -> bool:
```

Comment:
`is_relative_to` was causing circular import issues because it is also used in `validation.py`. I moved it to `validation.py` to prevent an error.

*question:* maybe, it makes sense to place `is_relative_to` to some other file? If you have ideas, please, share.